tasklist/client/src/modules/mutations/useCompleteTask.tsx

```diff
@@ -25,7 +25,15 @@ function useCompleteTask() {
         return response.json();
       }
 
-      throw error ?? new Error('Could not complete task');
+      if (error !== null || error !== undefined) {
+        const errorMessage =
+          error.response instanceof Response
```

Comment:
You don't need to use `instanceof`, that's why we have the `variant` inside the error object

Reply:
I misunderstood the purpose of the variant. I’ve removed the instanceof check accordingly.